### PR TITLE
static pages

### PR DESCRIPTION
**Pull request by nguyenthanhluanFramgia**

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -32,6 +32,7 @@ gem 'sdoc', '~> 0.4.0', group: :doc
 # Use Capistrano for deployment
```
Comment (hellovietnam93) [inline Gemfile]: xóa tất cả remark

Comment (nguyenthanhluanFramgia) [inline Gemfile]: updated

File: Gemfile [hunk 2]
```diff
@@ -1,47 +1,23 @@
 source 'https://rubygems.org'
 
-
-# Bundle edge Rails instead: gem 'rails', github: 'rails/rails'
 gem 'rails', '4.2.5'
```
Comment (hellovietnam93) [inline Gemfile]: uisng ""

File: app/assets/javascripts/static_pages.coffee [hunk 3]
```diff
@@ -0,0 +1,3 @@
+# Place all the behaviors and hooks related to the matching controller here.
```
Comment (hellovietnam93) [inline app/assets/javascripts/static_pages.coffee]: remove this file

File: app/assets/stylesheets/custom.css.scss [hunk 4]
```diff
@@ -0,0 +1,254 @@
+@import "bootstrap-sprockets";
+@import "bootstrap";
+
+/* mixins, variables, etc. */
+
+$gray-medium-light: #eaeaea;
+
+@mixin box_sizing {
+  -moz-box-sizing:    border-box;
```
Comment (hellovietnam93) [inline app/assets/stylesheets/custom.css.scss]: using only one space after ::

File: app/helpers/application_helper.rb [hunk 5]
```diff
@@ -1,2 +1,10 @@
 module ApplicationHelper
+  def full_title(page_title = '')
```
Comment (hellovietnam93) [inline app/helpers/application_helper.rb]: remove ()

File: app/helpers/application_helper.rb [hunk 6]
```diff
@@ -1,2 +1,10 @@
...
+    base_title = "FELS"
```
Comment (hellovietnam93) [inline app/helpers/application_helper.rb]: I18n for "FELS"

File: app/helpers/static_pages_helper.rb [hunk 7]
```diff
@@ -0,0 +1,2 @@
+module StaticPagesHelper
```
Comment (hellovietnam93) [inline app/helpers/static_pages_helper.rb]: remove this file

File: app/views/layouts/_footer.html.erb [hunk 8]
```diff
@@ -0,0 +1,10 @@
+<footer class="footer">
+  <small>
+   
+  </small>
+  <nav>
+    <ul>
+      
+    </ul>
+  </nav>
+</footer>
```
Comment (hellovietnam93) [inline app/views/layouts/_footer.html.erb]: lack of end file

File: app/views/static_pages/about.html.erb [hunk 9]
```diff
@@ -0,0 +1,3 @@
+<% provide(:title, 'About') %>
+<h1>StaticPages#about</h1>
```
Comment (hellovietnam93) [inline app/views/static_pages/about.html.erb]: I18n for StaticPages#about

File: app/assets/stylesheets/custom.css.scss [hunk 10]
```diff
@@ -0,0 +1,254 @@
...
+/* mixins, variables, etc. */
+
+$gray-medium-light: #eaeaea;
+
+@mixin box_sizing {
+  -moz-box-sizing: border-box;
+  -webkit-box-sizing: border-box;
+  box-sizing: border-box;
+}
+
```
Comment (hellovietnam93) [inline app/assets/stylesheets/custom.css.scss]: remove this line

File: app/assets/stylesheets/custom.css.scss [hunk 11]
```diff
@@ -0,0 +1,254 @@
...
+.center {
+  text-align: center;
+}
+
+.center h1 {
+  margin-bottom: 10px;
+}
+
+
```
Comment (hellovietnam93) [inline app/assets/stylesheets/custom.css.scss]: remove this line

File: app/assets/stylesheets/custom.css.scss [hunk 12]
```diff
@@ -0,0 +1,254 @@
...
+  text-align: center;
+  font-weight: normal;
+  color: #777;
+}
+
+p {
+  font-size: 1.1em;
+  line-height: 1.7em;
+}
+
```
Comment (hellovietnam93) [inline app/assets/stylesheets/custom.css.scss]: only one blank line to divide 2 block

File: app/assets/stylesheets/static_pages.scss [hunk 13]
```diff
@@ -0,0 +1,3 @@
+// Place all the styles related to the StaticPages controller here.
```
Comment (hellovietnam93) [inline app/assets/stylesheets/static_pages.scss]: remove this file

File: app/helpers/application_helper.rb [hunk 14]
```diff
@@ -1,2 +1,10 @@
 module ApplicationHelper
+  def full_title page_title = ''
```
Comment (hellovietnam93) [inline app/helpers/application_helper.rb]: "" instead of ''

File: app/helpers/application_helper.rb [hunk 15]
```diff
@@ -1,2 +1,10 @@
...
+    base_title = t("application.name")
```
Comment (hellovietnam93) [inline app/helpers/application_helper.rb]: at here don't need to use ()
only 
```
base_title = t "application.name"
```

File: app/views/layouts/_footer.html.erb [hunk 16]
```diff
@@ -0,0 +1,5 @@
+<footer class="footer">
+  <small>
+    <%= t "footer.me" %>
+  </small>
+</footer>
```
Comment (hellovietnam93) [inline app/views/layouts/_footer.html.erb]: lack of end file

File: app/views/layouts/_header.html.erb [hunk 17]
```diff
@@ -0,0 +1,13 @@
+<header class="navbar navbar-fixed-top navbar-inverse">
+  <div class="container">
+    <%= link_to t("application.name"), root_path, id: "logo" %>
+    <nav>
+      <ul class="nav navbar-nav navbar-right">
+        <li><%= link_to "Home", root_path %></li>
+        <li><%= link_to "Help", help_path %></li>
+        <li><%= link_to "About", about_path %></li>
+
```
Comment (hellovietnam93) [inline app/views/layouts/_header.html.erb]: remove this line

File: app/views/layouts/_header.html.erb [hunk 18]
```diff
@@ -0,0 +1,13 @@
+<header class="navbar navbar-fixed-top navbar-inverse">
+  <div class="container">
+    <%= link_to t("application.name"), root_path, id: "logo" %>
+    <nav>
+      <ul class="nav navbar-nav navbar-right">
+        <li><%= link_to "Home", root_path %></li>
```
Comment (hellovietnam93) [inline app/views/layouts/_header.html.erb]: I18n for "Home"

File: app/views/layouts/_shim.html.erb [hunk 19]
```diff
@@ -0,0 +1,4 @@
+<!--[if lt IE 9]>
+  <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/r29/html5.min.js">
+  </script>
+<![endif]-->
```
Comment (hellovietnam93) [inline app/views/layouts/_shim.html.erb]: end file

File: app/views/layouts/application.html.erb [hunk 20]
```diff
@@ -1,14 +1,34 @@
...
-  <%= csrf_meta_tags %>
-</head>
-<body>
-
-<%= yield %>
-
-</body>
+  <head>
+    <title><%= full_title(yield(:title)) %></title>
+    <%= stylesheet_link_tag 'application', media: 'all',
```
Comment (hellovietnam93) [inline app/views/layouts/application.html.erb]: using ""

File: app/views/layouts/application.html.erb [hunk 21]
```diff
@@ -1,14 +1,34 @@
...
+                                           'data-turbolinks-track' => true %>
```
Comment (hellovietnam93) [inline app/views/layouts/application.html.erb]: ident, one tab with line above

File: app/views/layouts/application.html.erb [hunk 22]
```diff
@@ -1,14 +1,34 @@
...
+    <%= csrf_meta_tags %>
+    <%= render 'layouts/shim' %>
+  </head>
+  <body>
+    <header class="navbar navbar-fixed-top navbar-inverse">
+      <div class="container">
+        <%= link_to t("application.name"), '#', id: "logo" %>
+        <nav>
+          <ul class="nav navbar-nav navbar-right">
+            <li><%= link_to "Home",   '#' %></li>
```
Comment (hellovietnam93) [inline app/views/layouts/application.html.erb]: one space after ```,```

File: config/database.yml.example [hunk 23]
```diff
@@ -1,25 +0,0 @@
-# SQLite version 3.x
```
Comment (hellovietnam93) [inline config/database.yml.example]: keep this file and clone a new file database.yml from this file

File: config/routes.rb [hunk 24]
```diff
@@ -1,4 +1,8 @@
 Rails.application.routes.draw do
+  root             'static_pages#home'
```
Comment (hellovietnam93) [inline config/routes.rb]: using "", and only one sapce

File: app/views/static_pages/home.html.erb [hunk 25]
```diff
@@ -0,0 +1,2 @@
+<% provide title: t("home") %>
+
```
Comment (hellovietnam93) [inline app/views/static_pages/home.html.erb]: remove this line

File: config/routes.rb [hunk 26]
```diff
@@ -1,56 +1,5 @@
...
-  #   resources :photos, concerns: :toggleable
-
-  # Example resource route within a namespace:
-  #   namespace :admin do
-  #     # Directs /admin/products/* to Admin::ProductsController
-  #     # (app/controllers/admin/products_controller.rb)
-  #     resources :products
-  #   end
+  root "static_pages#home"
+  get 'help' => "static_pages#help"
```
Comment (hellovietnam93) [inline config/routes.rb]: using ""

File: app/views/layouts/application.html.erb [hunk 27]
```diff
@@ -1,14 +1,22 @@
...
+      "data-turbolinks-track": true %>
+    <%= javascript_include_tag "application", "data-turbolinks-track": true %>
+    <%= csrf_meta_tags %>
+    <%= render "layouts/shim" %>
+  </head>
+  <body>
+    <%= render "layouts/header" %>
+    <div class="container">
+      <% flash.each do |message_type, message| %>
+        <%= content_tag(:div, message, class: "alert alert-#{message_type}") %>
```
Comment (hellovietnam93) [inline app/views/layouts/application.html.erb]: remove ()

**Pull request merged**